datacenter/ucp/2.1/guides/admin/upgrade/upgrade-offline.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
I think this will require a bit more work, since this is what I'm seeing right now

<img width="703" alt="screen shot 2017-02-17 at 15 36 19" src="https://cloud.githubusercontent.com/assets/1525937/23087167/f2546978-f526-11e6-934d-3b77a46923ed.png">

Since this change looks good, do you mind submitting a PR containing just the upgrade offline article with the necessary change to the `_data/toc.yaml` file, but keep reusing the `ddc_url_list.html component`?

Then we can work on another PR to separate the UCP and DTR offline tar files.